TicTacToe2/Math/Vec2.cs

```diff
@@ -66,26 +63,22 @@ public Vec2 SubtractY(int y)
         }
 
         // 7. Subtract int values x and y from the vector
-        public Vec2 SubtractXY(int x, int y)
+        public Vec2 Subtract(int x, int y)
         {
             return new Vec2(X - x, Y - y);
         }
 
         // 8. Subtract another Vec2 from the vector
-        public Vec2 SubtractAnother(Vec2 One, Vec2 Two)
+        public Vec2 Subtract(Vec2 V)
         {
-            int newX = One.X - Two.X;
-            int newY = One.Y - Two.Y;
-            Vec2 Three = new Vec2(newX, newY);
-            return Three;
+            return new Vec2(V.X - X, V.Y - Y);
         }
 
         // 9. Copy the vector
-        public Vec2(Vec2 other)
+        public Vec2 Copy(Vec2 V)
         {
-            X = other.X;
-            Y = other.Y;
-         }
+            return new Vec2(V.X, V.Y);
```

Comment:
I'm looking for a function that would allow me to write code like this
```c#
var a = new Vec2(1, 3);
var b = a.Copy(); // b is now a vector with value (1, 3)
```